### DISCUSSION TITLE
Does symfony/filesystem contain all of the methods in webmozart/path-util?

### DISCUSSION THREAD

**Discussion by romulasry** [symfony/symfony#44740] [Q&A]
> We could review symfony/filesystem and see if it really does contain all of the methods in path-util; last time I checked, it didn't, but that was a while ago.

https://github.com/drush-ops/drush/issues/4935#issuecomment-997106669

**Comment by stof** [comment]
Well, I suggest that you perform the comparison (I don't have the answer ready for you, because I don't know `webmozart/path-util`).

Btw, looking at the source of this in the drush issue tracker, this does not seem to be the right question to ask. The question should rather be "does symfony/filesystem contain of the Path methods that drush needs?" And the answer for that needs to come from someone knowing drush (seeing what method exist in the Path class is easy without prior knowledge of Symfony, but the part about drush needs requires some drush knowledge)

**Comment by greg-1-anderson** [comment]
Yes, I agree this should be handled in the Drush queue. I just didn't have time to look the other day.
